### DISCUSSION TITLE
Translateable select option labels

### DISCUSSION THREAD

**Discussion by paulhaem** [pimcore/pimcore#14242] [Ideas]
When using a select / multiselect field you can enter a `Display Name` and a `Value`. It would be nice to have instead of a single `Display Name` a column for each configured language / locale. As of now the only way to translate select options in the frontend is using Option Providers or Shared Translations. This adds quite the code and management overhead. It would also be a nice addition to the admin interface as it is by default multilanguage. It could display an according translation for a select option if matched with the logged-in users language and give the content creator some more comfort when selecting options inside a select or multiselect field.

What do you think?

**Comment by kingjia90** [comment]
> . It could display an according translation for a select option if matched with the logged-in users language and give the content creator some more comfort when selecting options inside a select or multiselect field.

Sounds similar to what we did on https://github.com/pimcore/pimcore/pull/14074, please have a look to see if it may help.

Did i get it right or the idea is to change in the UI to somehow display all the translations of each select option?

**Reply by paulhaem** [reply]
This works already fine when using an OptionProvider for select / multiselect fields and having a localized field as the key. The logged-in user then sees the select options in his configured language if available.

The idea was to add this behaviour without having to write a OptionProvider every time you want a select with translateable option labels.
